### PR TITLE
use setters instead constructor

### PR DESCRIPTION
**Pull request by KoNekoD**
https://app.shortcut.com/systeme/story/16825/move-communitymemberinvitationconsumer-to-community-backend-service

### CODE REVIEW

File: src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php [hunk 1]
```diff
@@ -6,13 +6,95 @@
...
-    public function __construct(
-        public readonly int $registeredUserDomainAccountUserId,
-        public readonly int $registeredUserDomainAccountContactId,
-        public readonly string $registeredUserDomainAccountLocale,
-        public readonly string $ownerSenderAddress,
-        public readonly string $ownerSenderName,
-        public readonly string $ownerRouteLocale,
-        public readonly ?string $confirmationToken,
-    ) {}
+    public int $registeredUserDomainAccountUserId;
```
Comment (nkylinenko) [inline src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:9]: another fields use term 'owner' - lets change this one the same way

Comment (nkylinenko) [reply src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:9]: also we could make it shorter (the class name has registeredUserDomainAccount words - it defines the context)
my suggestion is just $ownerId

File: src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php [hunk 2]
```diff
@@ -6,13 +6,95 @@
...
+    public int $registeredUserDomainAccountContactId;
```
Comment (nkylinenko) [inline src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:10]: this one could be just $contactId

File: src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php (hunk 1 above)
Comment (KoNekoD) [reply src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:9]: Do you realize we're getting two owners? One owner is me, invited him, the other is the user domain account of the guy I invited.

Comment (KoNekoD) [reply src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:9]: image

Comment (KoNekoD) [reply src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:9]: Are you sure it won't cause a problem? If it's okay, I'll change it, but I might forget it's "that guy" and not "me".

File: src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php (hunk 2 above)
Comment (KoNekoD) [reply src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:10]: So it's "my" contact, right?

File: src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php [hunk 3]
```diff
@@ -6,13 +6,95 @@
...
+    public string $registeredUserDomainAccountLocale;
```
[the remark this answers is not in the document]
Comment (KoNekoD) [reply src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:11]: How do I know whose locale it is? Mine or "that guy's"?

File: src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php (hunk 1 above)
Comment (KoNekoD) [reply src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:9]: $registeredUserDomainAccountUserId
$registeredUserDomainAccountContactId
$registeredUserDomainAccountLocale

Comment (nkylinenko) [reply src/Monolith/RegisterUserDomainAccount/RegisterUserDomainAccountOutputDTO.php:9]: ok I get it - looks not obvious

**Pull request merged**